tests/nosetests/pyanaconda_tests/network_test.py

```diff
@@ -47,11 +47,13 @@ def is_valid_hostname_test(self):
         self.assertFalse(network.is_valid_hostname(
             "section.may.not.end.with.hyphen-")[0])
 
-        self.assertTrue(network.is_valid_hostname("0-0.")[0])
-        self.assertTrue(network.is_valid_hostname("0.")[0])
+        #self.assertTrue(network.is_valid_hostname("0-0.")[0])
+        #self.assertTrue(network.is_valid_hostname("0.")[0])
```

Comment:
I am not sure what this actually tests. It could be that ``0`` is the minimal viable hostname component and thus also minimal hostname, and ``0-0`` a minimal variant that contains a hyphen. But I have seen the specs for that literally a few minutes ago, so I don't trust myself with that.

@rvykydal, you added this half a decade ago, so if you have any advice, I'd be very grateful. But if you have no idea any more, that's ok too :-)

Reply:
Sorry I don't remember. I'd guess the dot at the end is the main point of these checks but your guess may be true as well. I would remove it.